docs/amber_script.md

```diff
@@ -156,6 +145,29 @@ is to run no optimization passes.
 
 #### Bindings
 
+##### Framebuffer Formats
+ * R32G32B32A32_UINT
+
+Bind a provided framebuffer.
+
+```
+  FRAMEBUFFER <width> <height>
+  FRAMEBUFFER <width> <height> <fb_format>
+```
```

Comment:
We have to decide if FRAMEBUFFER is a new resource allocation or is an attribute of an existing buffer saying "it's an image, and it will be used as a framebuffer".   
In Vulkan it's more the latter, where a VkFramebuffer wraps around an image view (which points to the underlying image storage), and also has a format attached to it.

So I would think of FRAMEBUFFER as a named type, like

     FRAMEBUFFER <decl-name> <width> <height>

e.g.
    FRAMEBUFFER fb 640 480

See suggested modification in examples on how to use this.